packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -29,104 +29,108 @@ const StyledInputAsSelect = styled(InputElement)<ComboboxProps>(props =>
 );
 /* eslint-enable */
 
-const Item: React.FC<ItemProps> = ({item, index, getItemProps, highlightedIndex, optionTemplate, inGroup}) => {
-  const UIDSeed = useUIDSeed();
-  return (
-    <ComboboxListboxOption
-      {...getItemProps({item, index})}
-      highlighted={highlightedIndex === index}
-      key={UIDSeed(item)}
-      variant={inGroup ? 'groupOption' : 'default'}
-    >
-      {optionTemplate ? optionTemplate(item) : item}
-    </ComboboxListboxOption>
-  );
-};
+const Item = React.forwardRef<HTMLDivElement, ItemProps>(
+  ({item, index, getItemProps, highlightedIndex, optionTemplate, inGroup}, ref) => {
+    const UIDSeed = useUIDSeed();
+    return (
+      <ComboboxListboxOption
+        {...getItemProps({item, index})}
+        highlighted={highlightedIndex === index}
+        key={UIDSeed(item)}
+        variant={inGroup ? 'groupOption' : 'default'}
+        ref={ref}
+      >
+        {optionTemplate ? optionTemplate(item) : item}
+      </ComboboxListboxOption>
+    );
+  }
+);
 
-const Items: React.FC<ItemsProps> = ({items, getItemProps, highlightedIndex, optionTemplate}) => {
-  return (
-    <>
-      {items.map((item, index) => (
-        <Item
-          // eslint-disable-next-line react/no-array-index-key
-          key={index}
-          item={item}
-          index={index}
-          getItemProps={getItemProps}
-          highlightedIndex={highlightedIndex}
-          optionTemplate={optionTemplate}
-        />
-      ))}
-    </>
-  );
-};
+const Items = React.forwardRef<HTMLDivElement, ItemsProps>(
+  ({items, getItemProps, highlightedIndex, optionTemplate}, ref) => {
+    return (
+      <>
+        {items.map((item, index) => (
+          <Item
+            // eslint-disable-next-line react/no-array-index-key
+            key={index}
+            item={item}
+            index={index}
+            getItemProps={getItemProps}
+            highlightedIndex={highlightedIndex}
+            optionTemplate={optionTemplate}
+            ref={ref}
```

Comment:
In this case, you're passing the same ref to several elements (each item in the map). I don't think we should forwardRef here